src/_path.h

```diff
@@ -432,6 +432,7 @@ void point_in_path_collection(double x,
                                       subtrans(1, 1),
                                       subtrans(0, 2),
                                       subtrans(1, 2));
+            trans *= master_transform;
```

Comment:
It's one of two bugfixes.  This is that the transform was wrong (this line accidentally disappeared in the C++ refactor), and the other is that the results were being stored in a temporary array rather than the array returned to the caller.